adserver/urls.py

```diff
@@ -18,4 +18,6 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('placement/', include('placement.urls')),
```

Comment:
Where is 'placement.urls' defined?? Is there an app directory called 'placement'? I'm pretty sure not... You need to wait for @kieranuche3 and @haileyxc to merge in there changes (which I think they have), then you will see what the app directory is called.